java/arcs/core/data/Schema.kt

```diff
@@ -19,7 +19,7 @@ import arcs.core.type.Type
 typealias Refinement = (data: RawEntity) -> Boolean
 
 /** Returns true if the RawEntity data matches the query predicate (given a query argument)*/
-typealias Query = (data: RawEntity, queryArg: Any) -> Boolean
+typealias Query = (data: RawEntity, queryArgs: Any) -> Boolean
```

Comment:
did you not want to make this `vararg` anymore?

Reply:
Sorry, I thought you didn't get back to me about better ways to write this and so was continuing down the path towards using a single data class holding all the arguments, as I believe this is the closes that Kotlin gets to an anonymous record.